ft(rate-limiting-advanced): allow 'cluster' strategy with 'sync_rate … (#5025)
Kong/docs.konghq.com: app/_hub/kong-inc/rate-limiting-advanced/_index.md

        In DB-less and hybrid modes, the `cluster` config strategy
        is not supported. From `3.0.0.0` onwards, Kong disallows
        the plugin enablement if strategy is `cluster` and `sync_rate` is `-1`

>         with DB-less or hybrid mode. From `3.2.0.0` onward, please either
>         use a different strategy or set `sync_rate` as `-1`.

Reviewer: ```suggestion
        with DB-less or hybrid mode. From `3.2.0.0` onward, please
        use a different strategy or set `sync_rate` to `-1`.
```
